public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -87,11 +88,13 @@ const getStyles = (theme: GrafanaTheme2, placeholder: string) => {
 };
 
 const MonacoQueryField = (props: Props) => {
-  // we need only one instance of `overrideServices` during the lifetime of the react component
+  // we need only one instance of `overrideSerices` during the lifetime of the react component
```

Comment:
```suggestion
  // we need only one instance of `overrideServices` during the lifetime of the react component
```

Reply:
Should have been fixed in https://github.com/grafana/grafana/pull/60172/commits/73e283a17f7a4e0610b9ba212a136cfdcf189da5 already 😁